apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/NewProposalForm.constants.ts

```diff
@@ -0,0 +1,82 @@
+import { ProposalType } from './NewProposalForm.types';
+
+export const proposalTypeOptions = [
```

Comment:
```suggestion
export const PROPOSAL_TYPE_OPTIONS = [
```
Constants should be uppercased, change all of them in this file.